recipes-kernel/linux/linux-yocto_5.10.bbappend

```diff
@@ -0,0 +1,40 @@
+LICENSE = "MIT"
```

Comment:
Hi @preetam-reddy,

Regarding `KERNEL_DTBDEST` variable, here are my findings:
- This variable is used to specify the installation location of `.dtb` or `.dtbo` files within the image (i.e., where they are installed in the root filesystem). By default, they are placed in the `image/boot` directory (inside the image). You can use `KERNEL_DTBDEST` to customize the subfolder under `image` where these files are placed. However, it does not affect where the files are deployed to deploy dir (`build/tmp/deploy/images/rzg2l-sbc ): https://github.com/yoctoproject/poky/blob/scarthgap/meta/classes-recipe/kernel-devicetree.bbclass#L90. 

  ```
  rvcpoc@f99762f651a9:~/01_docker-shared/tien/yocto-scarthgap/build$ MACHINE=rzg2l-sbc bitbake linux-yocto -e | grep ^D=
  D="/home/rvcpoc/01_docker-shared/tien/yocto-scarthgap/build/tmp/work/rzg2l_sbc-poky-linux/linux-yocto/5.10.184+git/image"
  ```
  Default behaviour is still to deploy all the dtb files to `deploydir`: https://github.com/yoctoproject/poky/blob/scarthgap/meta/classes-recipe/kernel-devicetree.bbclass#L103. 

- Below is an example of how I modified the linux-yocto recipe to change the installation path for .dtb files:

  ```
  KERNEL_DTBDEST = "${KERNEL_IMAGEDEST}/dtb"
  ```
  
  ```
  $ MACHINE=rzg2l-sbc bitbake linux-yocto -e | grep KERNEL_IMAGEDEST=
  KERNEL_IMAGEDEST="boot"
  ```
  ![image](https://github.com/user-attachments/assets/6b474664-95f9-4e6d-b5a1-7b88dea369ee)
  ![image](https://github.com/user-attachments/assets/0cb16623-3d99-4e42-bc1c-8c8e8e5e49a2)
  
  The additional subdirectories like `renesas` or `overlays` appear because I defined the specific locations for the .dtb files within the KERNEL_DEVICETREE variables.
  
=> Therefore, if we want to customize the deployment (i.e., files placed in build/tmp/deploy/images), we need to keep  this append file.

Reply:
> This file is not needed in scarthgap. for device tree use the following in the main file:
> 
> ```
> # Supported device tree and device tree overlays
> KERNEL_DEVICETREE = " \
>         renesas/rzpi.dtb \
> "
> 
> KERNEL_DEVICETREE:append:rzg2l-sbc = " \
>         renesas/overlays/rzpi-can.dtbo \
>         renesas/overlays/rzpi-ext-i2c.dtbo \
>         renesas/overlays/rzpi-ext-spi.dtbo \
>         renesas/overlays/rzpi-dsi.dtbo \
>         renesas/overlays/rzpi-ov5640.dtbo \
> "
> ```

If we append all `dtbo` files into KERNEL_DEVICETREE variable, the poky will install all these files to `/boot`, which can clutter the `/boot` directory. So that's why `KERNEL_DEVICETREE_OVERLAYS` is include in this case and need one task to install to `/boot/overlays` folder.

```
rvcpoc@f99762f651a9:~/01_docker-shared/tien/yocto-scarthgap/build/tmp/deploy/images/rzg2l-sbc$ tree rootfs/boot/
rootfs/boot/
├── EFI
│   └── BOOT
│       ├── bootaa64.efi
│       └── grub.cfg
├── Image -> Image-5.10.184-cip36-yocto-standard
├── Image-5.10.184-cip36-yocto-standard
├── Readme.md
├── rzpi-can.dtbo
├── rzpi-dsi.dtbo
├── rzpi.dtb
├── rzpi-ext-i2c.dtbo
├── rzpi-ext-spi.dtbo
├── rzpi-ov5640.dtbo
└── uEnv.txt
```